challenges-webapp/src/main/webapp/skin/less/challenges.less

```diff
@@ -267,6 +267,30 @@
 .emptyChallenges {
   min-height: ~"calc(100vh - 100px)" ;
 }
+@media (max-width : @maxMobileWidth) {
+  .challenge-column {
+    width: 100%;
+    
+    }
+}
+@media (min-width : @minTabletWidth) and (max-width: @maxLargeTabletWidth) {
+  .challenge-column {
+    width: 50%;
+    
+    }
+}
+@media (min-width : @minDesktop) and (max-width: 1365px) {
+  .challenge-column {
+    width: 25%;
+    
+    }
+}
+@media (min-width : 1366px) {
+  .challenge-column {
+    width: 17%;
+    
+    }
+}
```

Comment:
Using `col-` classes, we don't need those CSS changes anymore.

Reply:
col- does not meet our needs, I need to display the number of challenges indicated (4 challenges in the first row) on the width of the whole interface